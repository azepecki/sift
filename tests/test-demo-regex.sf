def int main() {
	str regex =  "rk(0|1|2|3|4|5|6|7|8|9)*@columbia.edu";
	str valid_email = "rk3142@columbia.edu";
	str invalid_email = "rk3142";
        

	if (reg_test(valid_email, regex)) {
		str result_str = str_add(valid_email, " is valid");
		print(result_str);
	} else {
		print((str_add(valid_email, " is an invalid website url")));
	}


	return 0;

}