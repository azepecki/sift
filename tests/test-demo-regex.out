rk3142@columbia.edu is valid
